src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -258,6 +259,397 @@ def test_when_entity_ref_prefix_type_invalid_then_debug_log_printed(
 )
 
 
+class BatchFirstMethodTest:
```

Comment:
[Consider][Minor] Just something to explain what this is.
```suggestion

# This is a generic test-runner fixture that allows generic testing of OpenAssetIO style methods.
# Specifically, it allows testing of our "convenience" methods in all their forms, including the batch
# signatures they are all based upon.
class BatchFirstMethodTest:
```

Reply:
Good shout. Added a somewhat different docstring in 883d4f1f